package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "govuk-prototype-kit",
   "description": "Rapidly create HTML prototypes of GOV.UK services",
-  "version": "12.1.0",
```

Comment:
why are we changing the version number here? We normally do that in a release PR not in a fix/feature PR

Reply:
Ah, that was for testing, thanks for catching.